src/Cronofy/UpsertEventRequestBuilder.cs

```diff
@@ -578,6 +589,19 @@ public UpsertEventRequestBuilder RemoveAttendee(string email, string displayName
             return this;
         }
 
+        /// <summary>
+        /// Sets the event as having no start or end.
+        /// </summary>
+        /// <returns>
+        /// A reference to the modified builder.
+        /// </returns>
+        public UpsertEventRequestBuilder NoStartOrEnd()
```

Comment:
The reason I did it this way round rather than checking for if start/end are null is that we'd be losing some unintentional validation where the program will blow up if someone attempted to make an event without a start / end

Writing this out though I realise that unintentional validation probably isn't something we should protect. Intentional validation or it's just something we don't care about